app/scripts/modules/dcos/dcos.settings.ts

```diff
@@ -3,7 +3,7 @@ import { IProviderSettings, SETTINGS } from '@spinnaker/core';
 export interface IDcosProviderSettings extends IProviderSettings {
   defaults: {
     account?: string;
-    proxy?: string;
+    dcosCluster?: string;
```

Comment:
Yes, it's used in the create server group command builder, it allows an account to default to a specific cluster.